lib/erl/src/thrift_compact_protocol.erl

```diff
@@ -355,7 +356,7 @@ read(This0, i64) ->
 read(This0, double) ->
   {This1, Bytes} = read_data(This0, 8),
   case Bytes of
-    {ok, <<Val:64/float-signed-big, _/binary>>} -> {This1, {ok, Val}};
+    {ok, <<Val:64/float-signed-little, _/binary>>} -> {This1, {ok, Val}};
```

Comment:
@jeking3 thanks, good poinit. I've checked the [doc](http://erlang.org/doc/programming_examples/bit_syntax.html) but couldn't find any `double`.

> The type can be integer, float, or binary.

So I'll replace the other part with float and also unify the order.